build/version.sh

```diff
@@ -1,14 +1,17 @@
 #!/bin/bash
-version=$(git describe --always --tag --abbrev=0)
+version=$(npx semantic-release --dry-run | grep "next release version is " | sed -nE 's/(.*?(\bnext\srelease\sversion\sis\s)(.*?))$/\3/p')
```

Comment:
this assumes everyone who uses this has `npx` installed on their machine? Adding some documentation around that might be good or using default `npm`? If that's possible though.

Reply:
`semantic-release` is in package.json so i can also run as `./node_modules/.bin/semantic-release`. I confirmed that `npx` ran successfully in travis and focused only on that use-case ... the script is meant for travis deploy scenarios only - any advise?

Reply:
Hmm yeah I guess it is not a big issue in this case then. Apparently Travis provides `npx` by default. The only downside would be someone trying to run this script locally (for whatever reason) and not having `npx` installed. But they could always open an issue...